src/python/grpcio/tests/interop/methods.py

```diff
@@ -305,6 +307,30 @@ def _oauth2_auth_token(stub, args):
         'expected to find oauth scope "%s" in received "%s"' %
             (response.oauth_scope, args.oauth_scope))
 
+def _jwt_token_creds(stub, args):
```

Comment:
btw, for _oauth2_auth_token interop test (above), I am not sure if that test behaves completely up to the spec.  The idea of that test is to test that you can use a oauth2 token obtained out of band (but the test just seems to be using the googleCredentials directly).

I think for oauth2 token test, you need to grab the actual oauth2 token  and then create callCredentials from that specific token (basically https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Auth/GoogleGrpcCredentials.cs#L68). Then you use those callCredentials to make a call and check if everything was successful.